WalletWasabi.Gui/Config.cs

```diff
@@ -84,6 +86,24 @@ public class Config : ConfigBase
 		[JsonConverter(typeof(EndPointJsonConverter), Constants.DefaultRegTestBitcoinP2pPort)]
 		public EndPoint RegTestBitcoinP2pEndPoint { get; internal set; } = new IPEndPoint(IPAddress.Loopback, Constants.DefaultRegTestBitcoinP2pPort);
 
+		[DefaultValue(false)]
+		[JsonProperty(PropertyName = "JsonRpcServerEnabled", DefaultValueHandling = DefaultValueHandling.Populate)]
+		public bool JsonRpcServerEnabled { get; internal set; }
```

Comment:
Shouldn't we remove this? How to open multiple wallets when it's enabled?

Reply:
I think the RPC should be disabled by default and only those advanced users how really need it and know what they are doing can enabled it.

By the moment it works with the open wallet and doesn't support multiple wallets.